content/content_test.go

```diff
@@ -33,34 +33,43 @@ func TestContentParseOK(t *testing.T) {
 	con, err := content.ParseRuleContentDir("../tests/content/ok/")
 	helpers.FailOnError(t, err)
 
-	rule1Content, exists := con["rule1"]
+	rule1Content, exists := con.Rules["rule1"]
 	assert.True(t, exists, "'rule1' content is missing")
 
 	_, exists = rule1Content.ErrorKeys["err_key"]
 	assert.True(t, exists, "'err_key' error content is missing")
 }
 
+// TestContentParseOKNoContent checks that parsing content when there is no rule
+// content available, but the file structure is otherwise okay, succeeds.
+func TestContentParseOKNoContent(t *testing.T) {
+	con, err := content.ParseRuleContentDir("../tests/content/ok_no_content/")
+	helpers.FailOnError(t, err)
+
+	assert.Equal(t, 0, len(con.Rules))
```

Comment:
It would be better to use assert.Empty here

Reply:
@Sergey1011010 Good point, didn't know it existed.